pkg/gateway/lb/handler/grpc/handler.go

```diff
@@ -765,6 +788,18 @@ func (s *server) Insert(ctx context.Context, req *payload.Insert_Request) (ce *p
 		Uuid: uuid,
 		Ips:  make([]string, 0, s.replica),
 	}
+	if req.GetConfig().GetTimestamp() == 0 {
```

Comment:
I'm not sure but is that safe to call `GetTimestamp` without nil check of `req.GetConfig()`?

Reply:
yes, it's safe, because its generated by protobuf if GetConfig returns nil GetTimestamp return zero
FYI: https://github.com/vdaas/vald/blob/feature/gateway-agent%2Fgrpc-status-refactor/apis/grpc/v1/payload/payload.pb.go#L1207-L1212

Reply:
if we directly access to Timestamp like below it caused panic
`req.GetConfig().Timestamp`